package.json

```diff
@@ -84,14 +89,23 @@
       "stylelint-no-unsupported-browser-features"
     ],
     "rules": {
+      "string-quotes": "single",
+      "color-hex-length": "long",
```

Comment:
@kopepasah 
If this comes through from the Wordpress Standards, we should just deactivate it like this:
`"color-hex-length": null`

It doesn't make sense to apply this sort of rules in a scenario where we're always writing preprocessor code that gets minified and that gets the CSS compiled to using the short whenever possible.

In the Wordpress environment I think that comes through as a micro-optimisation because not all CSS files get minified through a build process.